include/DREAM/Equations/RunawayFluid.hpp

```diff
@@ -160,6 +162,8 @@ namespace DREAM {
 
         void Rebuild(const real_t);
         void GridRebuilt();
+        bool GetextrapolateDreicer()
```

Comment:
Could you change the name of this method to ``GetExtrapolateDreicer``, so that it is consistent with the naming of other methods in the code?